build-tools/packages/build-cli/src/handlers/checkFunctions.ts

```diff
@@ -446,6 +437,64 @@ export const checkPolicy: StateHandlerFunction = async (
 	return true;
 };
 
+/**
+ * Checks that a release branch exists.
+ *
+ * @param state - The current state machine state.
+ * @param machine - The state machine.
+ * @param testMode - Set to true to run function in test mode.
+ * @param log - A logger that the function can use for logging.
+ * @param data - An object with handler-specific contextual data.
+ * @returns True if the state was handled; false otherwise.
+ */
+export const checkAssertTagging: StateHandlerFunction = async (
+	state: MachineState,
+	machine: Machine<unknown>,
+	testMode: boolean,
+	log: CommandLogger,
+	data: FluidReleaseStateHandlerData,
+): Promise<boolean> => {
+	if (testMode) return true;
+
+	const { context, releaseGroup, shouldCheckPolicy } = data;
+	assert(context !== undefined, "Context is undefined.");
+	assert(releaseGroup !== undefined, "Release group is undefined.");
+
+	if (shouldCheckPolicy === true) {
+		if (getPolicyRunDefault(releaseGroup, context.originalBranchName)) {
```

Comment:
isn't this check backwards. Its warning if you do the default right?

```suggestion
		if (!getPolicyRunDefault(releaseGroup, context.originalBranchName)) {
```

Reply:
You're totally right. Fixed.